.gitlab-ci.yml

```diff
@@ -80,7 +80,7 @@ test:backward-compat:
   stage: test
   image: ubuntu:22.04
   before_script:
-    - apt update && apt install -yyq clang cmake git make pkg-config liblmdb++-dev libboost-dev libboost-log-dev libssl-dev libarchive-dev libdbus-1-dev curl dbus stunnel4 tinyproxy-bin netcat
+    - apt update && apt install -yyq ccache clang cmake git make pkg-config liblmdb++-dev libboost-dev libboost-log-dev libssl-dev libarchive-dev libdbus-1-dev curl dbus stunnel4 tinyproxy-bin netcat
```

Comment:
This is going to require more changes in our CI to work with a persistent cache. We don't use the shared runners, we have private ones that are provisioned on every run, therefore sharing a cache is a bit complicated. I'm not sure if this is worth the effort, since the runners are quite beefy, and the client is about as big as it's going to get now.

Reply:
Fair enough. It's up to you if you want me to remove ccache from the apt install!

Reply:
I guess we could leave it here just to make sure it works. We need to make sure we enable it then.